index.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-undef */
```

Comment:
- [ ] please avoid turning off the linters in your project, linters are used to optimize your code and make it more readable, therefore we advise you to keep them turned on.